app/utils/humanize-duration.js

```diff
@@ -21,15 +21,17 @@ export default function humanizeDuration(duration, seconds = false) {
     return seconds ? '0h 0m 0s' : '0h 0m'
   }
 
+  let prefix = duration.asMinutes() < 0 ? '-' : ''
```

Comment:
nitpick: wouldn't `asSeconds` be more correct? (What are negative durations used for, anyway?)

Reply:
Maybe `.milliseconds()` would be best.. Negative durations are used e.g. for negative overtime-credits

Reply:
As discussed: leaving out `asMinutes` and relying on default bahivour of moment should work, too. If not, it might be `asMilliseconds`.